src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureStreamingForm.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { InputLabel, SelectHorizontal, FormSection } from '~core/Fields';
+import { filterUserSelection } from '~core/SingleUserPicker';
+import UserPickerWithSearch from '~core/UserPickerWithSearch';
+import { useMembersSubscription } from '~data/generated';
+import { ExpenditureEndDateTypes } from '~pages/ExpenditurePage/types';
+
+import { supRenderAvatar } from '../Recipient/Recipient';
+
+import { Props } from './ExpenditureSettings';
+import styles from './ExpenditureSettings.css';
+
+export const MSG = defineMessages({
+  to: {
+    id: 'dashboard.ExpenditurePage.ExpenditureStreamingForm.to',
```

Comment:
Same here, this id and all ids below are missing .ExpenditureSettings: `dashboard.ExpenditurePage.ExpenditureSettings.ExpenditureStreamingForm.to`

Reply:
also it is fixed